internal/kibana/fleet.go

```diff
@@ -74,7 +86,8 @@ func (cfg FleetConfig) Flags() []string {
 		// TO-DO: remove all code to calculate the fleet-server policy, because it's inferred by the fleet-server
 		return []string{
 			"--force",
-			"--fleet-server-es", fmt.Sprintf("http://%s@%s:%d", cfg.ElasticsearchCredentials, cfg.ElasticsearchURI, cfg.ElasticsearchPort),
```

Comment:
If we are not going to need ElasticsearchCredentials anymore, then let's remove it from the struct

Reply:
This is still needed for the other flag permutations like enrollment